gui/app/api/index.js

```diff
@@ -1,14 +1,20 @@
+let parse = (res) => {
+    if(res.status === 200)
+        return res.json();
+    else
+    if(res.status === 400)
+        throw new Error("Input error, server failed to URL decode bibliographicRecordId");
```

Comment:
Da parse nu håndtere flere forskellige requests  så er en status 400 nok ikke lig med 
server failed to URL decode bibliographicRecordId. 
Der udover så vil det nok være rart hvis Fejl texten fra serveren kan ses et sted.